### ISSUE TITLE
Consider updating SplashScreen to use DevExpress capabilities

### ISSUE THREAD

**Issue by msevestre** [Open-Systems-Pharmacology/PK-Sim#608]
This would remove dependencies on VisualBasic runtimes and would allow for a nicer splash screen

**Comment by PavelBal** [comment]
And, hopefully, a splash screen that will not block the view while it is loading ;)

**Comment by msevestre** [comment]
Yes. I d also like to see the open systems pharmacology logo somewhere

**Comment by UlrichSi** [comment]
Let's hope for a positive experience. That seeing the logo for the time it takes to load a larger project (e.g., 10 simulations, 20 PI tasks) by double-clicking its file name in the Windows explorer will be joy and not grief.

**Comment by rwmcintosh** [comment]
@msevestre @Yuri05 @PavelBal 

I think this is obsolete now. 

Splash screen runs in a separate UI thread from the loading thread
Logo is present in the splash
Microsoft.VisualBasic.ApplicationServices is still used today, but not sure this warrants a change on its own.

**Comment by Yuri05** [comment]
let's close then :)

**Comment by rwmcintosh** [comment]
Closing as obsolete. If I missed anything, reopen.
